docs/wiki/installation/cli-flags.md

```diff
@@ -131,7 +129,7 @@ By default the watchdog monitors extensions for improper shutdown, but NOT for p
 
 `--table_delay=0`
 
-Add a microsecond delay between multiple table calls (when a table is used in a JOIN). A `200` microsecond delay will trade about 20% additional time for a reduced 5% CPU utilization.
+Add a millisecond delay between multiple table calls (when a table is used in a JOIN). A `200` millisecond delay will trade about 20% additional time for a reduced 5% CPU utilization.
```

Comment:
Seems it's milliseconds:
https://github.com/osquery/osquery/blob/2e3495837d4fe4db3554c6ef76494bc86c74c099/osquery/process/process.h#L217-L220